.gitignore

```diff
@@ -23,3 +23,11 @@ storage-layouts/proposed*
 
 # IDE
 .idea
+
+
+.anchor
```

Comment:
nit: can you move these into the appropriate commented sections above? maybe make a new one for solana